src/lib/mdns/minimal/records/tests/TestResourceRecordPtr.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include <mdns/minimal/records/Ptr.h>
+
+#include <support/TestUtils.h>
+
+#include <nlunit-test.h>
+
+namespace {
+
+using namespace chip;
+using namespace mdns::Minimal;
+
+void TestPtrResourceRecord(nlTestSuite * inSuite, void * inContext)
+{
+    uint8_t headerBuffer[HeaderRef::kSizeBytes];
+    uint8_t dataBuffer[128];
+
+    const QNamePart kName[] = { "some", "test", "local" };
+    const QNamePart kPtr[]  = { "ptr", "addr" };
+
+    HeaderRef header(headerBuffer);
+
+    BufBound output(dataBuffer, sizeof(dataBuffer));
+    PtrResourceRecord record(kName, kPtr);
+
+    record.SetTtl(123);
+
+    header.Clear();
+
+    const uint8_t expectedOutput[] = {
+        4, 's', 'o', 'm', 'e',      // QNAME part: some
+        4, 't', 'e', 's', 't',      // QNAME part: test
+        5, 'l', 'o', 'c', 'a', 'l', // QNAME part: local
+        0,                          // QNAME ends
+        0, 12,                      // QType PTR
+        0, 1,                       // QClass IN
+        0, 0,   0,   123,           // TTL
+        0, 10,                      // data size - size for IPv4
```

Comment:
Not sure what you mean by "size for IPv4"

Reply:
I mean "I copied and pasted this code from the IPv4 unit tetst and I was too distracted to change the comments".

Obviously :)

Changed.